task/build-vm-image/0.1/build-vm-image.yaml

```diff
@@ -148,6 +154,9 @@ spec:
         echo "$BUILD_DIR"
         ssh -v $SSH_ARGS "$SSH_HOST"  mkdir -p "$BUILD_DIR/workspaces" "$BUILD_DIR/scripts" "$BUILD_DIR/tmp" "$BUILD_DIR/tekton-results" "$BUILD_DIR/entitlement"
 
+        # ensure that a config toml file is present in case one is not provided
+        touch /var/workdir/source/$CONFIG_TOML_FILE
```

Comment:
If the user does have a config file but messes up the file path, their error will be silently ignored. Let's at least log a warning?

Reply:
What do you mean? If they mess up the file path then the config file will be created and used. I will print out the config that we are using after the file is touched.

Reply:
If I have a `cool-config.toml` that actually sets some options and I forget to override the default `config.toml`, the task will create a completely empty config and I'll be wondering why my config is being ignored

Reply:
How should we cover that? I added a cat of the toml file, do you think that should be sufficient?

Reply:
That's probably fine. Maybe also log a big WARNING when we create a previously non-existent file

Alternatively, the option could be smarter
* Empty string by default
  * But if `config.toml` exists use it anyway, otherwise create an empty config
* If not empty string, don't create an empty config. The user really wants their config file to be used

Reply:
Resolved in the latest revision.